src/make_trade.ts

```diff
@@ -1,12 +1,21 @@
+import {
+  OrderKind,
+  Order,
+  signOrder,
+  SigningScheme,
+  domain,
+} from "@gnosis.pm/gp-v2-contracts";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { HardhatEthersHelpers } from "@nomiclabs/hardhat-ethers/types";
 import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20.json";
 import { TokenInfo, TokenList } from "@uniswap/token-lists";
-import { BigNumber, Contract } from "ethers";
+import { BigNumber, Contract, ethers } from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
```suggestion
import { BigNumber, Contract } from "ethers";
import { ethers } from "hardhat";
import { HardhatRuntimeEnvironment } from "hardhat/types";
```

I suggest using the `hardhat` wrapped `ethers` as it configures the provide and signer for you.

Reply:
**edit**: Looks like you can't do this for tasks.